.github/workflows/main.yml

```diff
@@ -0,0 +1,123 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  windows:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        gmakeconfig:
+          - debug
+          #- profile
+          #- release
+        gmakeplatform:
+          - 64
+        ocamlversion:
+          - 4.13.x
+
+    steps:
+      - name: Checkout ocaml-orx
+        uses: actions/checkout@v2
+        with:
+          path: ocaml-orx
+
+      - name: Checkout orx
+        uses: actions/checkout@v2
+        with:
+          repository: orx/orx
+          path: orx
+
+      - name: orx setup
+        working-directory: orx
+        run: ./setup.bat
+
+      - name: orx build (gmake)
+        working-directory: orx/code/build/windows/gmake
+        run: make config=${{ matrix.gmakeconfig }}${{ matrix.gmakeplatform }}
+
+      - name: Set ORX environment
+        run: echo "ORX=${{ github.workspace }}/orx/code" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf8 -Append
+
+      - name: Setup OCaml
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ${{ matrix.ocamlversion }}
```

Comment:
It should work as expected by doing the following:

```yml
opam-local-packages: ocaml-orx/*.opam
```

https://github.com/ocaml/setup-ocaml/blob/master/examples.md#using-glob-patterns-to-filter-local-packages

Reply:
Makes sense - I'll test now!